odl/__init__.py

```diff
@@ -43,7 +43,7 @@
 from . import deform
 from . import ufunc_ops
 from . import datasets
-
+from . import contrib
```

Comment:
Second comment after reading more:

It seems like you need to rebase this onto master since this is far behind the current master